src/swap/SwapScreen.tsx

```diff
@@ -251,25 +255,30 @@ export function SwapScreen() {
     navigate(Screens.WebViewScreen, { uri: SWAP_LEARN_MORE })
   }
 
+  const edges: Edge[] | undefined = showDrawerTopNav ? undefined : ['bottom']
+
   return (
-    <SafeAreaView style={styles.safeAreaContainer}>
-      <DrawerTopBar
-        middleElement={
-          <View style={styles.headerContainer}>
-            <Text style={headerStyles.headerTitle}>{t('swapScreen.title')}</Text>
-            {exchangeRate && fromToken && toToken && (
-              <Text
-                style={[headerStyles.headerSubTitle, fetchingSwapQuote ? styles.mutedHeader : {}]}
-              >
-                {`1 ${fromToken.symbol} ≈ ${new BigNumber(exchangeRate).toFormat(
-                  5,
-                  BigNumber.ROUND_DOWN
-                )} ${toToken.symbol}`}
-              </Text>
-            )}
-          </View>
-        }
-      />
+    <SafeAreaView style={styles.safeAreaContainer} edges={edges}>
+      {showDrawerTopNav && (
+        <DrawerTopBar
+          testID={'SwapScreen/DrawerBar'}
+          middleElement={
+            <View style={styles.headerContainer}>
+              <Text style={headerStyles.headerTitle}>{t('swapScreen.title')}</Text>
+              {exchangeRate && fromToken && toToken && (
+                <Text
+                  style={[headerStyles.headerSubTitle, fetchingSwapQuote ? styles.mutedHeader : {}]}
+                >
+                  {`1 ${fromToken.symbol} ≈ ${new BigNumber(exchangeRate).toFormat(
+                    5,
+                    BigNumber.ROUND_DOWN
+                  )} ${toToken.symbol}`}
+                </Text>
+              )}
```

Comment:
Should we retain this on the header of the new screen?

Reply:
Updated the code so it keeps that subtitle